codegen/cmd.go

```diff
@@ -14,9 +19,8 @@ type Command struct {
 	// used to label k8s manifests
 	AppName string
 
-	// search protos recursively starting from this directory
-	// TODO: use anyvendor to support external imports
-	ProtoDir string
+	// config to vendor protos and other non-go files
+	AnyVendorConfig *ap_anyvendor.Imports
```

Comment:
can anyvendor be optional?
i would think you should be able to `anyvendor` first, then just set the ProtoDir to the anyvendor dir and have no issue